packages/web-components/src/components/ic-loading-indicator/ic-loading-indicator.stories.mdx

```diff
@@ -258,83 +259,87 @@ export const defaultArgTypes = {
   </Story>
 </Canvas>
 
-### Circular light
+### Circular dark mode
 
 <Canvas>
   <Story
-    name="Circular light"
+    name="Circular dark mode"
     parameters={{ backgrounds: { default: "dark" }, loki: { skip: true } }}
   >
     {(args) =>
       html`<div style="display: grid; gap: 50px;">
         <ic-loading-indicator
           label="Small"
           size="small"
-          appearance="light"
+          theme="dark"
         ></ic-loading-indicator>
         <ic-loading-indicator
           label="Default"
-          appearance="light"
+          theme="dark"
         ></ic-loading-indicator>
         <ic-loading-indicator
           label="Large"
           size="large"
-          appearance="light"
+          theme="dark"
+        ></ic-loading-indicator>
```

Comment:
There is already a default-sized loading indicator on line 276 so this one isn't needed